IntegrationTests/Tests/AccountTests.cs

```diff
@@ -288,4 +292,86 @@ public async Task Verify_email_return_200()
 
         response.StatusCode.Should().Be(HttpStatusCode.OK);
     }
+
+    [Fact]
+    public async Task Change_Password_Invalid_Model_State_Returns_Bad_Request()
+    {
+        var changePasswordDto = new ChangePasswordDto()
+        {
+            OldPassword = "oldPassword",
+            NewPassword = "newPassword"
+        };
+
+        var response =
+            await _client.PostAsync("/Identity/Account/change-password", JsonContent.Create(changePasswordDto));
+
+        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+    }
+
+    [Fact]
+    public async Task Change_Password_Valid_Old_Password_Changes_Password()
+    {
+        await ClearToken();
+        // Arrange
+        var newUser = new ApplicationUser()
+        {
+            Email = "change-password@app.com",
+            UserName = "change-password@app.com",
```

Comment:
i think it is sufficient to only specify email, but this is ofc not wrong either

Reply:
lmao